libs/platform/kernel/ebpf_platform_kernel.c

```diff
@@ -675,3 +675,9 @@ ebpf_guid_create(_Out_ GUID* new_guid)
     else
         return EBPF_OPERATION_NOT_SUPPORTED;
 }
+
+void
+ebpf_platform_printk(_In_z_ const char* output)
+{
+    EBPF_LOG_MESSAGE(EBPF_TRACELOG_LEVEL_INFO, EBPF_TRACELOG_KEYWORD_PRINTK, output);
```

Comment:
https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/wdm/nf-wdm-kdprintex as well?

Reply:
Alan and I discussed yesterday and tracelogging is closer to what Linux does.   Issue #795 tracks how to see live output.